modules/openshift-developer-cli-developer-setup.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/odo-architecture.adoc
+
+[id="developer-setup_{context}"]
+
+= Developer Setup
+
+With {odo-title} you can create and deploy application on OpenShift clusters from a terminal.  Code editors plug-ins use {odo-short} which allows users to interact with OpenShift clusters from their IDE terminals. Examples of plug-ins that use {odo-short}: VS Code Openshift Connector, Openshift Connector for Intellij, Codewind for Eclipse Che. 
```

Comment:
Should `Code editors plug-ins` be `Code editor plug-ins`? Something is off in the sentence as it is now.

Reply:
You're right. `plug-ins` are the subjects of the sentence.